Description: Fix lintian warning about hyphen used as minus sign
Author: Karl Ferdinand Ebert <kfebert@gmail.com>
Forwarded: not-needed

--- a/tmux.1
+++ b/tmux.1
@@ -465,7 +465,7 @@
 .Ql -
 may be followed by an offset, for example:
 .Bd -literal -offset indent
-select-window -t:+2
+select-window \-t:+2
 .Ed
 .Pp
 When dealing with a session that doesn't contain sequential window indexes,
@@ -521,13 +521,13 @@
 .Nm
 commands include:
 .Bd -literal -offset indent
-refresh-client -t/dev/ttyp2
+refresh-client \-t/dev/ttyp2
 
-rename-session -tfirst newname
+rename-session \-tfirst newname
 
-set-window-option -t:0 monitor-activity on
+set-window-option \-t:0 monitor-activity on
 
-new-window ; split-window -d
+new-window ; split-window \-d
 
 bind-key R source-file ~/.tmux.conf \e; \e
 	display-message "source-file done"
@@ -536,11 +536,11 @@
 Or from
 .Xr sh 1 :
 .Bd -literal -offset indent
-$ tmux kill-window -t :1
+$ tmux kill-window \-t :1
 
-$ tmux new-window \e; split-window -d
+$ tmux new-window \e; split-window \-d
 
-$ tmux new-session -d 'vi /etc/passwd' \e; split-window -d \e; attach
+$ tmux new-session \-d 'vi /etc/passwd' \e; split-window \-d \e; attach
 .Ed
 .Sh CLIENTS AND SESSIONS
 The
@@ -970,7 +970,7 @@
 .Pa /tmp
 as well as the paste buffer:
 .Bd -literal -offset indent
-bind-key -temacs-copy C-q copy-pipe "cat >/tmp/out"
+bind-key \-temacs-copy C-q copy-pipe "cat >/tmp/out"
 .Ed
 .Pp
 The paste buffer key pastes the first line from the top paste buffer on the
@@ -1150,7 +1150,7 @@
 and the result executed as a command.
 If
 .Ar template
-is not given, "detach-client -t '%%'" is used.
+is not given, "detach-client \-t '%%'" is used.
 For the meaning of the
 .Fl F
 flag, see the
@@ -1198,7 +1198,7 @@
 and the result executed as a command.
 If
 .Ar template
-is not given, "switch-client -t '%%'" is used.
+is not given, "switch-client \-t '%%'" is used.
 For the meaning of the
 .Fl F
 flag, see the
@@ -1245,7 +1245,7 @@
 Note that
 .Ql %%
 can be used and will be replaced with the session name.
-The default option if not specified is "switch-client -t '%%'".
+The default option if not specified is "switch-client \-t '%%'".
 If
 .Fl c
 is given, will override the default window command.
@@ -1288,7 +1288,7 @@
 and the result executed as a command.
 If
 .Ar template
-is not given, "select-window -t '%%'" is used.
+is not given, "select-window \-t '%%'" is used.
 For the meaning of the
 .Fl F
 flag, see the
@@ -1592,7 +1592,7 @@
 option only opens a new pipe if no previous pipe exists, allowing a pipe to
 be toggled with a single key, for example:
 .Bd -literal -offset indent
-bind-key C-p pipe-pane -o 'cat >>~/output.#I-#P'
+bind-key C-p pipe-pane \-o 'cat >>~/output.#I-#P'
 .Ed
 .It Xo Ic previous-layout
 .Op Fl t Ar target-window
@@ -2039,8 +2039,8 @@
 and be set to any string.
 For example
 .Bd -literal -offset indent
-$ tmux setw -q @foo "abc123"
-$ tmux showw -v @foo
+$ tmux setw \-q @foo "abc123"
+$ tmux showw \-v @foo
 abc123
 .Ed
 .Pp
@@ -2537,7 +2537,7 @@
 Examples are:
 .Bd -literal -offset indent
 #(sysctl vm.loadavg)
-#[fg=yellow,bold]#(apm -l)%%#[default] [#S]
+#[fg=yellow,bold]#(apm \-l)%%#[default] [#S]
 .Ed
 .Pp
 Where appropriate, special character sequences may be prefixed with a number to
@@ -2770,7 +2770,7 @@
 or with a terminal escape sequence.
 It may be switched off globally with:
 .Bd -literal -offset indent
-set-window-option -g automatic-rename off
+set-window-option \-g automatic-rename off
 .Ed
 .Pp
 .It Ic automatic-rename-format Ar format
@@ -3479,7 +3479,7 @@
 and the result executed as a command.
 If
 .Ar template
-is not given, "paste-buffer -b '%%'" is used.
+is not given, "paste-buffer \-b '%%'" is used.
 For the meaning of the
 .Fl F
 flag, see the
@@ -3827,22 +3827,22 @@
 .Pp
 Changing the default prefix key:
 .Bd -literal -offset indent
-set-option -g prefix C-a
+set-option \-g prefix C-a
 unbind-key C-b
 bind-key C-a send-prefix
 .Ed
 .Pp
 Turning the status line off, or changing its colour:
 .Bd -literal -offset indent
-set-option -g status off
-set-option -g status-style bg=blue
+set-option \-g status off
+set-option \-g status-style bg=blue
 .Ed
 .Pp
 Setting other options, such as the default command,
 or locking after 30 minutes of inactivity:
 .Bd -literal -offset indent
-set-option -g default-command "exec /bin/ksh"
-set-option -g lock-after-time 1800
+set-option \-g default-command "exec /bin/ksh"
+set-option \-g lock-after-time 1800
 .Ed
 .Pp
 Creating new key bindings:
